quantumdb-cli/src/main/java/io/quantumdb/cli/xml/XmlCreateTable.java

```diff
@@ -24,10 +24,13 @@ static XmlOperation convert(XmlElement element) {
 		operation.setTableName(element.getAttributes().get("tableName"));
 
 		for (XmlElement child : element.getChildren()) {
+			// "columns" tag is optional in Liquibase. In QuantumDB documentation and getting started both are used
 			if (child.getTag().equals("columns")) {
 				for (XmlElement subChild : child.getChildren()) {
 					operation.getColumns().add(XmlColumn.convert(subChild));
 				}
+			} else if (child.getTag().equals("column")) {
+				operation.getColumns().add(XmlColumn.convert(child));
```

Comment:
In the documentation of QuantumDB, the "columns" tag is used, in the getting started section it is left out.
Liquibase supports both, the "columns" tag is optional.

Reply:
There would be more to it than this to support an additional child tag. The XSD file would also need to be changed for instance. Additionally the aim was never to be 1-1 compatible with Liquibase, just to have a similar (easy) way of defining changesets. Not to say this will never change, but for now I'd prefer to keep this as is, as this change would also require us to deal with people mixing and matching `column` and `columns` (which I don't think we can even restrict in XSD). To keep this simple, for now I'd like to avoid changing the XSD without a good reason as this is a public interface.

Reply:
Ok, I removed this piece.
In the documentation under createTable I added the necessary tag.